### PR TITLE
Sara / Split about us on footer

### PR DESCRIPTION
**Pull request by sara-fs**
# Description

Summary of the change and/or issue that is fixed.

Changes:

-   fixes issue with item
-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Refactor code
-   [ ] Update translation text
-   [ ] Breaking change
-   [ ] Dependency update
-   [ ] This change requires a documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/about/index.js [hunk 1]
```diff
@@ -22,20 +121,102 @@ const About = () => {
...
+                                            'Jean-Yves has been an entrepreneur since the age of 20. From 1997 to 1999, he developed the algorithms that would become one of the world’s first trading platforms. He was granted a patent for his binary options trading system in 2007, and granted two more patents in 2011 for systems and methods that enable financial market speculation.',
+                                        )}
+                                    </Text>
+                                </div>
+                            </LeadershipWrapper>
+                        </ContentWrapper>
+                    )}
+                </StyledContainer>
+            </Background>
+            {active_tab === 'story' && <OurStory />}
```
Comment (mustofa-binary) [inline src/pages/about/index.js]: isnt this the same as above, we can combine them right?

File: src/pages/about/index.js [hunk 2]
```diff
@@ -1,19 +1,118 @@
...
+    return [active_tab, setTab]
+}
+const About = () => {
+    const [active_tab, setTab] = useTabState('story')
+    const is_story = active_tab === 'story'
+    const is_leadership = active_tab === 'leadership'
+    useEffect(() => {
+        const new_tab = getLocationHash() || 'story'
+        setTab(new_tab)
+    })
```
Comment (oskar-binary) [inline src/pages/about/index.js]: I think we only want to this to fire on mount?

```suggestion
    }, [])
```

Comment (oskar-binary) [reply src/pages/about/index.js]: oops didn't mean to approve it!

File: src/pages/about/index.js (hunk 1 above)
Comment (sara-fs) [reply src/pages/about/index.js]: Oh yes I changed it to: 
```
{is_story && <OurStory />}
            {is_leadership && <Leaders />}
```

File: src/pages/about/index.js (hunk 2 above)
Comment (oskar-binary) [reply src/pages/about/index.js]: actually nevermind, you can change it back to the old way as in this way the component is not re-rendered when you update the locationhash from the link in the footer.

**Pull request merged**